CodingDojo/server/src/main/java/com/codenjoy/dojo/services/PlayerGame.java

```diff
@@ -175,4 +180,13 @@ public void fireOnLevelChanged() {
         Player player = getPlayer();
         player.getEventListener().levelChanged(game.getProgress());
     }
+
+    public void setTeamId(int teamId) {
+        game.getPlayer().setTeamId(teamId);
+        getPlayer();
+    }
+
+    public int getTeamId() {
+        return game.getPlayer().getTeamId();
```

Comment:
what about `getPlayerTeamId()`?
do we want to leave with two getters for teamId property?
which one to use?